dev/tests/integration/testsuite/Magento/Catalog/Controller/ProductTest.php

```diff
@@ -11,6 +11,7 @@
 
 /**
  * @magentoAppIsolation enabled
+ * @magentoAppArea frontend
```

Comment:
Why do we need this fix?

Reply:
Trevis builds were failed.

Reply:
It is clear that Travis was failed
But I ask about cause of this failing
We can't change behavior in core code without significant cause

Reply:
Ok.